src/components/Spacer/Spacer.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+type Props = {
+  top: number;
+  right: number;
+  bottom: number;
+  left: number;
+};
+
+const Spacer = ({ top, right, bottom, left }: Props) => {
+  const paddingTop = top * 8;
+  const paddingRight = right * 8;
+  const paddingBottom = bottom * 8;
+  const paddingLeft = left * 8;
+
+  return (
+    <div
+      style={{
+        padding: `${paddingTop}px ${paddingRight}px ${paddingBottom}px ${paddingLeft}px`,
+      }}
```

Comment:
one last thing, can you make this styled-component styles?
```suggestion
      css={{
        padding: `${paddingTop}px ${paddingRight}px ${paddingBottom}px ${paddingLeft}px`,
      }}
```

also import 
```typescript
import 'styled-components/macro'
```

those two things do some CSS magic, like automatic [vendor prefixing](https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix).